prow/plugins.yaml

```diff
@@ -12,170 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+config_updater:
+  maps:
+    prow/config.yaml:
+      name: config
+    prow/plugins.yaml:
+      name: plugins
+
 plugins:
-  GoogleCloudPlatform/gke-application-security-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-cloud-sql-postgres-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-monitoring-tutorial:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-datadog-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-network-policy-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-enterprise-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-networking-demos:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-istio-gce-demo:            
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-rbac-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-istio-shared:               
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-rolling-updates-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-istio-telemetry-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-security-scenarios-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-istio-vpn-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-stateful-applications-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-logging-sinks-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-tracing-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-migration-to-containers:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-tracing-demo:
+  GoogleCloudPlatform:
```

Comment:
Is this safe to do? Do we receive webhooks for all repos under `GoogleCloudPlatform` (org scoped webhook)? If so this probably isn't acceptable.

Reply:
Nevermind, I just saw: https://github.com/GoogleCloudPlatform/oss-test-infra/pull/4#discussion_r261424809
> we receive webhooks for individual repos

Reply:
we only receive per-repo webhooks afaik

Reply:
If that isn't a certainty we should validate that before merging this so we don't start commenting (and trying to do more) in repos where Prow should not be enabled.

Reply:
(I don't think either me or @chrislovecnm has org-level webhook access, we only had touched repos we have access to)